sdk/keyvault/Azure.Security.KeyVault.Certificates/perf/Infrastructure/CertificatesScenarioBase.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Test.Perf;
+
+namespace Azure.Security.KeyVault.Certificates.Perf.Infrastructure
+{
+    public abstract class CertificatesScenarioBase<T> : PerfTest<T> where T : PerfOptions
+    {
+        private readonly Random _rand;
```

Comment:
Sure, but I don't see any reason to wait on that here. I can use it when ready. Or are you asking me to add it now?

I use random names because for Key Vault there's no way to know exactly when purged resources are truly purged. It can take minutes, so subsequent runs of this app (like when testing) can fail, which the perf test doesn't handle very well, BTW (it keeps running and printing lines of zeroes).